hook/mutate_persistentvolumeclaim.go

```diff
@@ -28,7 +28,7 @@ func PVCMutator(c client.Client, dec *admission.Decoder) http.Handler {
 	return &webhook.Admission{Handler: persistentVolumeClaimMutator{c, dec}}
 }
 
-// +kubebuilder:webhook:path=/pvc/mutate,mutating=true,failurePolicy=fail,groups="",resources=persistentvolumeclaims,verbs=create,versions=v1,name=pvc-hook.topolvm.cybozu.com
+// +kubebuilder:webhook:path=/pvc/mutate,mutating=true,failurePolicy=fail,matchPolicy="Equivalent",groups="",resources=persistentvolumeclaims,verbs=create,versions=v1,name=pvc-hook.topolvm.cybozu.com
```

Comment:
```suggestion
// +kubebuilder:webhook:path=/pvc/mutate,mutating=true,failurePolicy=fail,matchPolicy=equivalent,groups="",resources=persistentvolumeclaims,verbs=create,versions=v1,name=pvc-hook.topolvm.cybozu.com
```

I think it is good to uncapitalize the value and remove double-quotes just like other markers.

Reply:
Note: both of `Equivalent` and `equivalent` are valid.

https://github.com/kubernetes-sigs/controller-tools/blob/1c6d3844762fed588be5123e5b38ccbeb25d94e1/pkg/webhook/parser.go#L190-L201

Reply:
> I think it is good to use lowercase value and remove double-quotes just like other markers.

 I agree with it.